### DISCUSSION TITLE
塞内卡 | /cn/2019/08/seneca/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/yihui#1508] [Comments]
在一篇文章《现代生活压抑的五大原因》中看到一位古罗马先贤，叫塞内卡。我之前并不知道这个人，翻看了一下维基页面之后觉得跟我以前提过的《爱比克泰德》有些相似之处。他们都谈到了控制自己的内心，包括控制对财富的欲望。那篇文章引用了塞内卡的这句话：
[&hellip;] It is not the man who has too little that is poor, but the one who &hellip;

https://yihui.org/cn/2019/08/seneca/

**Comment by hilaily** [comment]
这两个人都是斯多葛学派的，这个学派也挺有意思，感觉他们的很多人生哲学挺适合当代人的
